### ISSUE TITLE
Configure the date sort options in the DLME app

### ISSUE THREAD

**Issue by corylown** [sul-dlss/dlme#1479]
- Configure ability to sort results by date in ascending order: `Sort by Date (old to new)`. Ascending order sort uses the minimum date value derived from cho_date_range_norm_isim for sorting. Field is likely to be `cho_date_norm_min_isi`
- Configure ability to sort results by date in descending order: `Sort by Date (new to old)`. Descending order uses the maximum date value derived from cho_date_range_norm_isim for sorting. Field is likely to be `cho_date_norm_max_isi`

Blocked by:

- #1477 

Relates to:

- #1470 
- #1476 
- #1478

Partially addresses:

- #1348

**Comment by corylown** [comment]
Can be QA'd if date sort is configured via admin/curator UI. Sort missing last is not yet implemented, but the work necessary for this feature is captured by another issue #1476
